### PR TITLE
[DOC] Write multi gpu training docs

### PR DESCRIPTION
**Pull request by eunwoosh**
Add document about multi GPU training at otx/docs/source/guide/get_started/quick_start_guide/cli_commands.rst

### CODE REVIEW

File: docs/source/guide/get_started/quick_start_guide/cli_commands.rst [hunk 1]
```diff
@@ -223,6 +223,9 @@ For example, that is how we can change the learning rate and the batch size for
     (otx) ...$ otx train SSD --train-data-roots <path/to/train/root> --val-data-roots <path/to/val/root> params --learning_parameters.batch_size 16 --learning_parameters.learning_rate 0.001
 
 
+As you can see from the arguments, the model can be trained using multiple GPUs. To do so, you simply need to specify a comma-separated list of GPU indices after the `--gpus` argument. The model will then be trained using distributed data parallel with the GPUs you have specified.
+
+Please note that multi-GPU training is currently not supported for the action task and semi/self-supervised learning. However, it is supported for all other tasks.
```
Comment (jaegukhyun) [inline docs/source/guide/get_started/quick_start_guide/cli_commands.rst]: How about adding we will support multi-GPU training in currently unsupported task?

**Review by jaegukhyun — COMMENTED** [review]
LGTM, I left a comment

File: docs/source/guide/get_started/quick_start_guide/cli_commands.rst (hunk 1 above)
Comment (sungmanc) [inline docs/source/guide/get_started/quick_start_guide/cli_commands.rst]: Regarding the @jaegukhyun 's comment, 
```suggestion
Please note that multi-GPU training is currently supported for all tasks except for action tasks and semi/self-supervised learning methods. We'll cover them in the near future.
```

**Review by supersoob — DISMISSED** [review]

**Review by sungmanc — DISMISSED** [review]

File: docs/source/guide/get_started/quick_start_guide/cli_commands.rst [hunk 2]
```diff
@@ -223,6 +223,9 @@ For example, that is how we can change the learning rate and the batch size for
     (otx) ...$ otx train SSD --train-data-roots <path/to/train/root> --val-data-roots <path/to/val/root> params --learning_parameters.batch_size 16 --learning_parameters.learning_rate 0.001
 
 
+As you can see from the arguments, the model can be trained using multiple GPUs. To do so, you simply need to specify a comma-separated list of GPU indices after the `--gpus` argument. The model will then be trained using distributed data parallel with the GPUs you have specified.
```
Comment (kprokofi) [inline docs/source/guide/get_started/quick_start_guide/cli_commands.rst]:
```suggestion
As can be seen from the parameters list, the model can be trained using multiple GPUs. To do so, you simply need to specify a comma-separated list of GPU indices after the `--gpus` argument. The model will then be trained using distributed data parallel with the GPUs you have specified.
```

File: docs/source/guide/get_started/quick_start_guide/cli_commands.rst (hunk 1 above)
Comment (kprokofi) [inline docs/source/guide/get_started/quick_start_guide/cli_commands.rst]:
```suggestion
.. note:

    Multi-GPU training is currently supported for all tasks except for action tasks and semi/self-supervised learning methods. We'll add support for them in the near future.
```

**Review by sungmanc — APPROVED** [review]

**Review by kprokofi — APPROVED** [review]

**Pull request merged**